packages/phoenix-ng/projects/phoenix-app/src/app/sections/lhcb/lhcb.component.ts

```diff
@@ -48,14 +48,36 @@ export class LHCbComponent implements OnInit {
     const configuration: Configuration = {
       eventDataLoader: new PhoenixLoader(),
       presetViews: [
-        new PresetView('Right View', [0, 0, 6000], 'right-cube'),
-        new PresetView('Center View', [-500, 1000, 0], 'top-cube'),
-        new PresetView('Left View', [0, 0, -6000], 'left-cube'),
+        new PresetView(
+          'Default View',
+          [-8000, 6000, 0],
+          [0, 0, 8000],
+          'perspective'
+        ),
+        new PresetView(
+          'Side View',
+          [-12000, 0, 12000],
+          [0, 0, 12000],
+          'right-cube'
+        ),
+        new PresetView(
+          'Vertex Locator View',
+          [-500, 1000, 0],
+          [0, 0, 0],
+          'top-cube'
+        ),
+        new PresetView('Front View', [0, 0, -6000], [0, 0, 0], 'left-cube'),
+        new PresetView(
+          'Top View',
+          [0, 16000, 12000],
+          [0, 0, 12000],
+          'top-cube'
+        ),
       ],
-      defaultView: [-18000, 0, 15000],
+      defaultView: [-8000, 6000, 0, 0, 0, 8000], // x,y,z of position followed by x,y,z of target
```

Comment:
The position and target are implicit here. The usage could be confusing for consumers. I think it's better to replace the array with `PresetView` and use it. Your thoughts?